{
  "messages": [
    {
      "error_id": "",
      "references": [
        {
          "column": 9,
          "file": "/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/SPMDIR-MUST/static/037-MPI-static-loop-array-bounds-local-yes.c",
          "line": 42
        },
        {
          "column": 25,
          "file": "/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/SPMDIR-MUST/static/037-MPI-static-loop-array-bounds-local-yes.c",
          "line": 45
        }
      ],
      "text": "Found get operation/call at results-20250730-140308/SPMDIR-MUST/static/037-MPI-static-loop-array-bounds-local-yes.c:42:9 which is in conflict with store operation/call at results-20250730-140308/SPMDIR-MUST/static/037-MPI-static-loop-array-bounds-local-yes.c:45:25 before release",
      "type": "Local Data Race"
    }
  ]
}
